app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name.downcase}", type: model_name) %></th>
```

Comment:
Do you want to use `model_name.param_key` (e.g. `GenericWork.model_name.param_key`)?